container-core/src/main/java/com/yahoo/container/handler/ClustersStatus.java

```diff
@@ -15,6 +15,8 @@
  * will survive reconfiguration events and inform other components even immediately after a reconfiguration
  * (where the true statue of clusters may not yet be available).
  *
+ * This is multithread safe.
```

Comment:
We already use the `javax.annotation.concurrent.ThreadSafe` annotation in 4 modules, which is easier to grep for than (hread safe or threadsafe or multithread safe or all misspellings)

Reply:
I can't imagine why anyone would grep for it